pkg/csi/cephfsdaemonset.go

```diff
@@ -48,6 +48,9 @@ var cephFSDaemonSetSpec = appsv1.DaemonSetSpec{
 		MatchLabels: cephfsDaemonsetLabels,
 	},
 	Template: corev1.PodTemplateSpec{
+		ObjectMeta: metav1.ObjectMeta{
```

Comment:
i haven't tried, all the labels present in the metadata need to be set in the selection, and here right? not just the defaults. Please check

Reply:
only one match is enough and selectors.matchLabels is immutable once set.